jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarModeProcessor.java

```diff
@@ -173,6 +168,28 @@ static JarType determineJarType(Path jarPath) throws IOException {
     }
   }
 
+  /**
+   * Computes the entrypoint for a standard jar in packaged mode.
+   *
+   * @param jarPath path to jar file
+   * @return list of {@link String} representing entrypoint
+   * @throws IOException if I/O error occurs when opening the jar file
+   */
+  static ImmutableList<String> computeEntrypointForPackagedStandard(Path jarPath)
+      throws IOException {
+    try (JarFile jarFile = new JarFile(jarPath.toFile())) {
+      String mainClass =
+          jarFile.getManifest().getMainAttributes().getValue(Attributes.Name.MAIN_CLASS);
+      if (mainClass == null) {
+        throw new IllegalArgumentException(
+            "`Main-Class:` attribute for an application main class not defined in the input Jar's "
```

Comment:
Not that I care, but just FYI, this [doc](https://docs.oracle.com/javase/tutorial/deployment/jar/basicsindex.html) stylizes the archive as "JAR." It stands for "Java™ Archive (JAR)" files.

Reply:
Oh nice, I will follow the convention in that case.